apps/web/src/components/organization-select.tsx

```diff
@@ -42,31 +42,45 @@ import {
   FormMessage,
 } from "./ui/form";
 import { Input } from "./ui/input";
+import { useToast } from "./ui/use-toast";
+
+
+
 
-type OrganizationForm = Omit<InsertOrganization, "owner">;
 
 export default function OrganizationSelect() {
   const pathname = usePathname();
-
   const organizationFromPathname = decodeURIComponent(
     // @ts-expect-error Since we are taking something from a pathname there has to be something
     pathname.split("/").at(2),
   );
-
   const router = useRouter();
+  const { toast } = useToast();
 
   const [isModalOpen, setIsModalOpen] = useState(false);
 
-  const { data: organizations, refetch } = useQuery({
+  useEffect(() => {
+    refetch();
+  }, [pathname]);
```

Comment:
What this is for?

Reply:
After the organization is deleted and the path is changed this component needs to refetch organizations, so it doesn't display the one that no longer exists.